server/routes/auth.routes.js

```diff
@@ -4,7 +4,7 @@ const router = express.Router();
 
 // login expects email/password
 // successful login returns email and a fake token (if we ever want to use it)
-router.post('/login', (req, res) => {
+router.post('/login', async (req, res) => {
```

Comment:
This route doesn't perform any asynchronous actions, so `async` is not needed here.